ui/shared/components/panel/variants/Annotations.jsx

```diff
@@ -110,10 +110,12 @@ const LOF_FILTER_MAP = {
   ANC_ALLELE: { title: 'Ancestral Allele', message: 'The alternate allele reverts the sequence back to the ancestral state' },
 }
 
-const getSvRegion = ({ chrom, endChrom, pos, end, liftedOverGenomeVersion, liftedOverPos }, divider) => {
+const getSvRegion = (
+  { chrom, endChrom, pos, end, liftedOverGenomeVersion, liftedOverPos }, divider, useLiftoverVersion,
+) => {
   const endOffset = endChrom ? 0 : end - pos
-  const start = liftedOverGenomeVersion === GENOME_VERSION_37 ? liftedOverPos : pos
-  return `${chrom}${divider || '-'}${start}-${start + endOffset}`
+  const start = liftedOverGenomeVersion === useLiftoverVersion ? liftedOverPos : pos
```

Comment:
I am concerned about what could happen if the `liftedOverGenomeVersion` is undefined in the variant and the `useLiftoverVersion` is also undefined.